src/Microsoft.ML.Transforms/CategoricalHashTransform.cs

```diff
@@ -120,6 +120,41 @@ public sealed class Arguments : TransformInputBase
 
         public const string UserName = "Categorical Hash Transform";
 
+        public static IDataTransform Create(IHostEnvironment env, IDataView input, int hashBits = 16, uint seed = 314489979,
```

Comment:
For managing these default values, const can added and used wherever the defaults needed.